### DISCUSSION TITLE
Problem with perimeters and infill not joining in overhangs

### DISCUSSION THREAD

**Discussion by neoxm3** [supermerill/SuperSlicer#3576] [Q&A]
Hi! 
First of all say that I am relatively new to super slicer, very happy with how it works on my machines with klipper. I have experience in cura and simplify3d, and ihave tried to change several things before asking for help but I have the following problem that I cannot solve, and that instead in cura for example, I do not have.

I have this piece, complex piece but prints relatively well:

https://i.ibb.co/y8phdfF/bd369393-3692-48d4-abe6-da6425b4033f.jpg

You can see at the top, in the recess for an embedded m5 screw, that the perimeters next to the infill, fill together perfectly, we have no gaps and everything apparently looks good.

But the problems are in the supported areas, the "overhangs", i dont know why, these areas They look much worse than in cura, the infill is very noticeable, like the first supported layer lacks material?

Here you can see it a little:
https://i.ibb.co/BKL5V3v/132ba52f-63f6-4788-b863-87bae5ec3bca.jpg

But in this other zone,  with much more support, the effect is much worse, the perimeters of the hole arent join to the infill:
https://i.ibb.co/LZ6cxtc/3213b0f4-004a-47b7-aa61-6425bfe4cf41.jpg

Also i dont know why the infill is not solid and the lines are so noticeable.

Someone can help me with some clue why i have this look so bad in the areas that are supported?

i attach in a file my printer settings.
[SuperSlicer_config_bundle.zip](https://github.com/supermerill/SuperSlicer/files/10419954/SuperSlicer_config_bundle.zip)

THANKS in advance!

**Comment by neophyl** [comment]
The best way to get help is to attach a copy of your project file.  File>save project as.  A 3mf project is snapshot, it contains the model, any placement or modifiers and a copy of the 3 profiles you are slicing with.  As opposed to a config bundle which contains every custom profile on your system, so who know which one you are slicing with.  We can also then see the slice preview and get a much better idea of what is happening.

On models like that its easy to redesign the part to add a single layer thick patch over any hole, that way the bottom is printed as one single bridging layer.  Then its easy to punch out the hole as its a single layer thick.

**Comment by psilofski** [comment]
What did you see at the preview tab? Does it show a reasonable amount of solid layers above the overhangs?
If not, then maybe you have a similar problem that I had a while ago, check in

Print Settings -> (Quality) No solid infill over: 

and change that value to zero. Then it should calculate solid layers
